core/frontend/src/extension/providers/ExtensionServiceClient.ts

```diff
@@ -0,0 +1,161 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
in hindsight, we should move this into its own repo/package after we release 3.2 since the extension CLI will need it as well once we bring it back. Nothing to do for now, just noting

Reply:
resolving this...